content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md

```diff
@@ -9,52 +9,61 @@ tags:
 
 {{< toc >}}
 
-The **Credentials > Local Groups** screen displays a list of groups configured on the screen. By default, built-in groups are hidden until you make them visible. 
+## Groups Screen
 
+The **Credentials > Local Groups** screen displays a list of groups configured on the screen. By default, built-in groups are hidden until you make them visible. 
 
-![GroupsListedSCALE](/images/SCALE/22.12/GroupsListedSCALE.png "Local Groups Hide Built-in Accounts") 
+{{< trueimage src="/images/SCALE/23.10/GroupsListedSCALE.png" alt="Local Groups Screen" id="1: Local Groups Screen" >}}
 
 To see built-in groups, click the **Show Built-In Groups** toggle. The toggle turns blue and all built-in groups display. Click the **Show Built-In Groups** toggle again to show only non-built-in groups on the system.
 
 The **Credentials > Local Groups** screen displays the **No groups** screen if no groups other than built-in groups are configured on the system.
 
-**Add** or **Add Groups** opens the **[Add Group](#add-group-screen)** configuration screen.
+**Add** opens the **[Add Group](#add-group-screen)** configuration screen.
 
-## Groups Details Screen
+Click the <span class="material-icons">expand_more</span> arrow or anywhere on a row to expand that group and show the group management buttons.
 
-The expanded view of each group includes details on that group and provides the option to edit members. Click the <span class="material-icons">expand_more</span> arrow to show the group details screen.
+{{< trueimage src="/images/SCALE/23.10/GroupsListedExpandedSCALE.png" alt="Expanded Group Screen" id="2: Expanded Group Screen" >}}
 
-![GroupsListedExpandedSCALE](/images/SCALE/22.12/GroupsListedExpandedSCALE.png "Local Group Details") 
-
-**Members** opens the **[Update Members](#update-members-screen)** screen. **Delete** opens a delete confirmation dialog.
+Use these buttons to add or remove group <i class="material-icons" aria-hidden="true" title="Members">group</i> [**Members**](#update-members-screen), <i class="material-icons" aria-hidden="true" title="Edit">edit</i> [**Edit**](#edit-group-screen) group configuration, or <i class="material-icons" aria-hidden="true" title="Delete">delete</i> **Delete** the group.
```

Comment:
same comment as in the other article, no need to mention the buttons or use the icons here.
```suggestion
Use [**Members**](#update-members-screen),  [**Edit**](#edit-group-screen) group configuration, or  **Delete** to add or remove a group .
```